content/en/docs/concepts/configuration/secret.md

```diff
@@ -172,16 +213,21 @@ spec:
 
 What will happen:
 
-* `username` secret is stored under `/etc/foo/my-group/my-username` file instead of `/etc/foo/username`.
-* `password` secret is not projected.
+* the `username` key from `mysecret` is available to the container at the path
+  `/etc/foo/my-group/my-username` file instead of at `/etc/foo/username`.
+* the `password` key from that Secret object is not projected.
 
 If `.spec.volumes[].secret.items` is used, only keys specified in `items` are projected.
-To consume all keys from the secret, all of them must be listed in the `items` field.
-All listed keys must exist in the corresponding secret. Otherwise, the volume is not created.
+To consume all keys from the Secret, all of them must be listed in the `items` field.
 
-#### Secret files permissions
+{{< note >}}
+If you list keys explicitly, then all listed keys must exist in the corresponding Secret.
+Otherwise, the volume is not created.
+{{< /note >}}
```

Comment:
Please avoid flooding the page with 'note' shortcode. We can treat all contents as noteworthy or else we won't document them.

Reply:
I wonder if the answer here is a new kind of callout with a more subtle highlight color (eg gray).

I want to call this out but only a little bit.

Reply:
Seems like you could make it a note or not.

Reply:
Callouts are useful for unexpected warnings, subtle hints, ... 
Since shortcodes are creating quite some issues for localization, I'd suggest we use them carefully.